src/ducks/apps/components/AppInstallation.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from 'react'
+
+import PermissionsList from './PermissionsList'
+import ReactMarkdownWrapper from '../../components/ReactMarkdownWrapper'
+
+import { getLocalizedAppProperty } from 'ducks/apps'
+import { translate } from 'cozy-ui/react/I18n'
+
+class AppInstallation extends Component {
+  installApp () {
+    this.setState({ error: null })
+    const { app, onSuccess } = this.props
+    this.props
+      .installApp(app.slug, app.type)
+      .then(() => {
+        onSuccess(app)
+      })
+      .catch()
```

Comment:
We don't catch anything? Or is this caught by the Redux store? I thinks either we need it and we have to process the error either we don't need it at all here.